app/mailers/consolidated_mailer.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+class ConsolidatedMailer < ApplicationMailer
+  def opening_reminder_email(course, user_email, items_hash)
+    @items_hash = items_hash
+    @recipient = user_email.user
```

Comment:
Why not just send in the `user` object into this method, rather than having to get `user.primary_email_record` in the consolidated job service?